### PR TITLE
DDF-3713 Add security exceptions for paths retrieved via System.getProperty

### PR DESCRIPTION
**Pull request by SmithJosh**
#### Who is reviewing it? 
@bakejeyner @blen-desta @brjeter @idperez 
#### Select relevant component teams: 
@codice/security 
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@coyotesqrl
@stustison
#### What are the relevant tickets?
[DDF-3713](https://codice.atlassian.net/browse/DDF-3713)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/ddf-common/src/main/resources/security/default.policy [hunk 1]
```diff
@@ -346,6 +346,36 @@ grant codeBase "file:/platform-parser-xml" {
...
+
+grant codeBase "file:/catalog-core-solrcommands/security-sts-attributequeryclaimshandler/catalog-confluence-source/org.eclipse.jetty.websocket.common" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}*", "read";
+}
+
+grant codeBase "file:/security-sts-attributequeryclaimshandler/security-sts-ldapclaimshandler" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}attributeMap.properties", "read";
+}
+
+grant codeBase "file:/security-servlet-session-expiry/org.eclipse.jetty.websocket.common" {
```
Comment (coyotesqrl) [inline distribution/ddf-common/src/main/resources/security/default.policy]: ❗️ Should find a way to collapse the two rows with `org.eclipse.jetty.websocket.common` to reduce startup time.

File: distribution/ddf-common/src/main/resources/security/default.policy [hunk 2]
```diff
@@ -346,6 +346,36 @@ grant codeBase "file:/platform-parser-xml" {
...
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}*", "read";
+}
+
+grant codeBase "file:/platform-solr-server-standalone" {
```
Comment (coyotesqrl) [inline distribution/ddf-common/src/main/resources/security/default.policy]: ❗️ This bundle is already referenced once in the policy file and the full set of permissions it needs (including this new one) are on the row for `grant codeBase "file:/catalog-core-standardframework"`.

Should remove this row, then move `platform-solr-server-standalone` from its current row to the standard framework row.

File: distribution/ddf-common/src/main/resources/security/default.policy [hunk 3]
```diff
@@ -346,6 +346,36 @@ grant codeBase "file:/platform-parser-xml" {
...
+grant codeBase "file:/catalog-core-localstorageprovider" {
+    permission java.io.FilePermission "${ddf.home.perm}content", "write";
+    permission java.io.FilePermission "${ddf.home.perm}content${/}-", "read, write";
+}
+
+grant codeBase "file:/platform-country-converter-local" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}fipsToIso.properties", "read";
+}
+
+grant codeBase "file:/org.ops4j.pax.web.pax-web-extender-war" {
```
[the remark this answers is not in the document]
Comment (SmithJosh) [reply distribution/ddf-common/src/main/resources/security/default.policy]: I added these permissions to the existing block

File: distribution/ddf-common/src/main/resources/security/default.policy (hunk 1 above)
Comment (SmithJosh) [reply distribution/ddf-common/src/main/resources/security/default.policy]: Did a simple merge of the two blocks

File: distribution/ddf-common/src/main/resources/security/default.policy (hunk 2 above)
Comment (SmithJosh) [reply distribution/ddf-common/src/main/resources/security/default.policy]: Done

**Comment by SmithJosh** [comment]
build now

**Pull request merged**